datasets/glossis/glossis.py

```diff
@@ -117,3 +128,21 @@
 
     # Wait for all the tasks to finish
     wait_gee_tasks(taskids)
+
+
+    # This should result in flowmap tiff files
+    flowmap_task_ids = []
+    flowmap_tiffs = []
+    for current_asset in current_assets:
+        flowmap_tiff = pathlib.Path(current_asset).with_suffix('.tif').name
+        flowmap_tiffs.append(flowmap_tiff)
```

Comment:
This is a bit implicit, this will be the names generated from the `export_flowmap` function? Good to introduce usage of pathlib. :)